content/kb/troubleshooting-websocket_handshake_error_with_unexpected_response_code_200.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Troubleshooting: Longhorn-UI: Error during WebSocket handshake: Unexpected response code: 200 #2265"
+author: Clark Hsu
+draft: false
+date: 2021-03-09
+catelogies:
+  - "longhorn-ui"
+---
+
+## Applicable versions
+
+Existing Longhorn versions < v1.1.0 upgrade to Longhorn >= v1.1.0.
+
+## Symptoms
+
+After upgraded Longhorn to version >= v1.1.0, and encounter the following:
```

Comment:
After upgrading Longhorn to verion >= v1.1.0, the following is encountered:

Reply:
changed